DeepSpeech.ipynb

```diff
@@ -616,7 +616,9 @@
     "    batch_x, batch_y, n_steps = batch_set.next_batch()\n",
     "\n",
     "    # Set batch_seq_len for the batch\n",
-    "    batch_seq_len = batch_x.shape[0] * [n_steps]\n",
+    "    multiple = tf.slice(tf.shape(batch_x),[0],[1])\n",
+    "    tile = tf.slice(tf.shape(batch_x),[1],[1])\n",
+    "    batch_seq_len = tf.tile(tile, multiple)\n",
```

Comment:
I mailed a reply to this but it isn't appearing, so here it is again:

OK, that's fair, but that comment still ought to be better I think. At the moment it's adding nothing. Even if it was something like 'prepare an array of sequence lengths for items in batch_x'.

Also, could it still not be something like:
```
batch_shape = tf.shape(batch_x)
batch_seq_length = batch_shape[0] * [batch_shape[1]]
```
?